red_black_cards.py

```diff
@@ -0,0 +1,44 @@
+#Problem Statement: "You are given a standard deck of cards with 26 black cards and 26 red cards. You're now going to play a game: 
+#in every round, you can take one of two actions: you can stop playing, or you can draw another card. If you draw another card and 
+#it's red, you get one dollar. If you draw another card and it's black, you lose one dollar. What is the expected value of this game?"
+
+import math
+
+try:
+    print("Input value cannot be cast to an int")
+    exit()
+
+try:
+import math
+
+try:
+    print("Input value cannot be cast to an int")
+    exit()
+
+try:
+    red = int(input('How many red cards are there?\n'))
+except:
+    print("Input value cannot be cast to an int")
+    exit()
+
+def expected_value(int: b, int: r) -> int:
```

Comment:
Still need to add a docstring. See [here](https://github.com/ThrunGroup/FastForest/blob/main/experiments/exp_utils.py#L12-L18) for correct format